src/ol/render/webgl/MixedGeometryBatch.js

```diff
@@ -519,18 +519,15 @@ class MixedGeometryBatch {
 
   /**
    * @param {Feature|RenderFeature} feature Feature
+   * @param {import("../../proj.js").TransformFunction} [projectionTransform] Projection transform.
    */
-  changeFeature(feature) {
+  changeFeature(feature, projectionTransform) {
     // the feature is not present in the batch; do not add it to avoid unexpected behaviors
     if (!this.uidToRef_.get(getUid(feature))) {
       return;
     }
     this.removeFeature(feature);
-    const geometry = feature.getGeometry();
-    if (!geometry) {
-      return;
-    }
-    this.addGeometry_(geometry, feature);
```

Comment:
Do you think you could keep the original approach which was to copy the code used in `addFeature` instead of simply calling this function? each function call has an overhead and this code might be run thousands/millions of times and it has a direct impact on the frame rate. Thanks!